buffer/src/main/java/io/netty/buffer/PooledUnsafeDirectByteBuf.java

```diff
@@ -61,7 +61,7 @@ void initUnpooled(PoolChunk<ByteBuffer> chunk, int length) {
     }
 
     private void initMemoryAddress() {
-        memoryAddress = PlatformDependent.directBufferAddress(memory) + offset;
+        memoryAddress = PlatformDependent.directBufferAddress(memory).add(offset);
```

Comment:
check if you can just do .memoryAddress?

Reply:
Sadly not, it's a package private method